### PR TITLE
Fixed Parse runtime exception for open-source MAPS.ME version.

### LINKED ISSUE #1990: iOS Runtime error
After setting up the project as per the stated steps for iOS in XCODE 7 I am facing this error on runtime 

**Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: ''applicationId' should not be nil.'**

### PR DESCRIPTION
**Pull request by deathbaba**
Should fix #1990

### CODE REVIEW

**Comment by deathbaba** [comment]
DUDE JTTP

**Comment by igrechuhin** [comment]
LGTM
@yunikkk PTAL

File: android/src/com/mapswithme/maps/MwmApplication.java [hunk 1]
```diff
@@ -228,7 +228,12 @@ private native void nativeInitPlatform(String apkPath, String storagePath, Strin
    */
   private void initParse()
   {
-    Parse.initialize(this, PrivateVariables.parseApplicationId(), PrivateVariables.parseClientKey());
+    // Do not initialize Parse in default open-source version.
+    final String paid = PrivateVariables.parseApplicationId();
```
Comment (yunikkk) [inline android/src/com/mapswithme/maps/MwmApplication.java]: Эм. Может, closed или internal?

**Comment by deathbaba** [comment]
@yunikkk fixed, merge please.

**Pull request merged**